ckanext/stadtzh_losdharvest/profiles.py

```diff
@@ -56,7 +56,7 @@
 license_cd_for_license = {
```

Comment:
what is a license cd? If this means code, could you call it `license_code_for_license` instead please?